### DISCUSSION TITLE
Works good on ROG Strix G513 IM

### DISCUSSION THREAD

**Discussion by greg262s** [seerge/g-helper#213] [General]
Asus should fire their marketing and programing "experts" for Armory Crate and hire YOU! Thank-you.

**Comment by greg262s** [comment]
I want to re-install Windows 11 in order to reduce the number of Asus partitions and recovery stuff.  What is the best way to get the necessary Asus drivers without all the crap?

**Reply by seerge** [reply]
I think all drivers from asus website are pretty much ok (at least for g14 model), except GPU/chipset ones that are a bit outdated. For your case it could be different tho :) 

For G-helper main thing is to have asus control interface installed, as it's communication layer between bios and the app.
